### DISCUSSION TITLE
Is there an official supported queueing service?

### DISCUSSION THREAD

**Discussion by 0xabrar** [vercel/community#344] [Help] [answered]
I want to be able to queue simple jobs on a delay or some simple way to fanout and run some job async. I see there used to be https://quirrel.dev/ to make using queues simple, but it no longer supports hosted versions (acquired by Netlify). Would really prefer not using SQS, so looking for alternatives.

**Comment by leerob** [comment, marked as answer]
**Edit: Update 2023**

Wanted to share an exciting update – we just released Vercel Cron Jobs!

Vercel Cron Jobs enable you to run scheduled jobs for automating backups and archiving, sending email and Slack notifications, and more. Cron Jobs can be used for any task you need to run on a schedule.

https://vercel.com/blog/cron-jobs
